persistence/test/ibc_test.go

```diff
@@ -0,0 +1,122 @@
+package test
+
+import (
+	"testing"
+
+	coreTypes "github.com/pokt-network/pocket/shared/core/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSetIBCStoreEntry(t *testing.T) {
```

Comment:
Please follow the same pattern elsewhere

```suggestion
func TestIBC_SetStoreEntry(t *testing.T) {
```